build-models.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Define the list of OpenAPI specifications and their corresponding API names and destination directories
+declare -A SPECS
+SPECS["serviceinfo"]="https://raw.githubusercontent.com/ga4gh-discovery/ga4gh-service-info/develop/service-info.yaml lib/src/serviceinfo/"
+SPECS["tes"]="https://raw.githubusercontent.com/ga4gh/task-execution-schemas/develop/openapi/task_execution_service.openapi.yaml lib/src/tes/"
+
+# Directory of the build script
+BUILD_SCRIPT="./script.sh"
```

Comment:
This does not seem to be available?

Reply:
I think I haven't pushed script.sh yet, will do it now.